lib/Runtime/Language/InlineCache.cpp

```diff
@@ -26,8 +26,6 @@ namespace Js
         // We may, however, have a flags cache (setter) change to add property cache.
         Assert(typeWithoutProperty == nullptr || !IsProto());
 
-        requestContext->RegisterAsScriptContextWithInlineCaches();
```

Comment:
Well.. that would be different then before in ThreadContext::ClearScriptContextCaches, where it would have cleared this cache before, but not now. Although it is just an heuristic to indicate whether a script context was used....?

It is also wrong that if threadContext->ClearInlineCaches() before, it would clear this script context, now it doesn't.